build/package/Dockerfile

```diff
@@ -19,6 +19,7 @@ FROM registry.access.redhat.com/ubi9-minimal@sha256:a7d837b00520a32502ada85ae339
 LABEL idmsvc-backend=backend
 # https://developers.redhat.com/articles/2022/05/31/your-go-application-fips-compliant
 ENV OPENSSL_FORCE_FIPS_MODE=1
+ENV SSL_CERT_DIR="/etc/ssl/certs:/etc/pki/tls/certs:/system/etc/security/cacerts:/cdapp/certs"
```

Comment:
Could you please add a comment explaining where the TLS CA certs are coming from (clowder operator populates the cdapp directory with trust roots).

Reply:
:+1: thanks mate! updated